compiler/IREmitter/sends.cc

```diff
@@ -96,7 +96,7 @@ llvm::Value *tryFinalMethodCall(MethodCallContext &mcctx) {
 
     // slow path: emit a call via the ruby vm
     builder.SetInsertPoint(slowFinalCall);
-    auto *slowPathResult = IREmitterHelpers::emitMethodCallViaRubyVM(mcctx);
+    auto *slowPathResult = tryNameBasedIntrinsic(mcctx);
```

Comment:
As discussed on slack yesterday, I think it's correct to leave this as-is; WDYT about instead adding a commit here explaining why this should be `emitMethodCallViaRubyVM`?

Reply:
Yup, will do!